resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -36,46 +36,57 @@
     language: bash
     program: |
 
-      IGV_FILES=()
-
+      # make folders with data
       {% for d in data %}
+        {% set species = d.species or 'other_data' %}
```

Comment:
@jkokosar I have added this line for files with unspecified spices and/or build information to be stored in a folder 'other_data'.